rust-runtime/aws-smithy-checksums/README.md

```diff
@@ -1,4 +1,4 @@
-# aws-smithy-checksum-callbacks
```

Comment:
I noticed the README was wrong so I updated it. LMK if you want me to remove this from this PR.